### DISCUSSION TITLE
Is there a way in api v3 to get all favored listing users like findAllListingFavoredBy in api v2?

### DISCUSSION THREAD

**Discussion by UUUEEELLL** [etsy/open-api#1017] [Ideas]
Hi, 
Is there a way in api v3 to get all favored listing users like findAllListingFavoredBy in api v2?

**Comment by Gareth064** [comment]
Hi @UUUEEELLL 

I'm afraid not.

There are no endpoints on V3 that allow you to get information about other users, with the exception of getUser which doesn't give you much of anything.
